src/Command/GuideCommand.php

```diff
@@ -182,12 +179,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             mkdir($dirName, 0777, true);
         }
         if (!file_put_contents($out, $content)) {
-            $style->getErrorStyle()->error(sprintf('Cannot write in "%s".', $out));
+            $stderr->error(sprintf('Cannot write in "%s".', $out));
 
             return self::FAILURE;
         }
 
-        $style->success(sprintf('Guide "%s" successfully created.', $file->getPathname()));
+        if (!$input->getOption('quiet')) {
+            $stderr->success(sprintf('Guide "%s" successfully created.', $file->getPathname()));
```

Comment:
```suggestion
            $style->success(sprintf('Guide "%s" successfully created.', $file->getPathname()));
```
Or if you prefer using `$output`, the `$style` variable can be removed and `$stderr` can be replaced by `$stderr = new SymfonyStyle($input, $output->getErrorOutput())`.

Reply:
I want stderr here